www/proxy-sso/src/opnsense/mvc/app/controllers/OPNsense/ProxySSO/Api/ServiceController.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+namespace OPNsense\ProxySSO\Api;
+
+use \OPNsense\Core\Backend;
+use \OPNsense\Core\Config;
+use \OPNsense\ProxySSO\ProxySSO;
+
+class ServiceController extends \OPNsense\Proxy\Api\ServiceController
```

Comment:
Is this base controller still an requirement?

Reply:
Yes.